src/concurrency/thread.rs

```diff
@@ -181,6 +181,41 @@ impl<'mir, 'tcx> Thread<'mir, 'tcx> {
     }
 }
 
+impl VisitMachineValues for Thread<'_, '_> {
+    fn visit_machine_values(&self, visit: &mut impl FnMut(&Operand<Provenance>)) {
+        let Thread { panic_payload, last_error, stack, .. } = self;
```

Comment:
My plan was to never use `..` here (since then we might add more fields and forget about visiting them) and instead explicitly do `field: _` for fields that do not visiting. RA can help with adding all those field names.

Reply:
`Thread` has a private field so I can add `field: _` for all the public fields but we can't get rid of the `..`.

Reply:
This is the file defining `Thread` so you should be able to access all private fields.

Reply:
Oh I meant `Frame` :facepalm:

Reply:
Yeah okay `Frame` will need a `..` with a comment explaining why.